server/apiCalls/getFilteredData.js

```diff
@@ -237,7 +248,10 @@ function _parseRounds({ roundInfos: koppsRoundInfos, courseCode, language, memoL
 const getFilteredData = async ({ courseCode, language, memoList }) => {
   const { body: koppsCourseDetails } = await koppsCourseData.getKoppsCourseData(courseCode, language)
   const { course: ladokCourse, rounds: ladokRounds } = await ladokApi.getCourseAndActiveRounds(courseCode, language)
+  const utbildningstillfalleUids = koppsCourseDetails.roundInfos.map(info => info.round.ladokUID)
 
+  // Just pick one
+  const examinationModules = await ladokApi.getExaminationModules(utbildningstillfalleUids[0], language)
```

Comment:
Wa are moving away from kopps data, and in this case we already have what we need om the course (utbildningsinstans), so no need to go the extra path through kurstillfällen here.